app/client/templates/elements/balance.js

```diff
@@ -28,13 +28,15 @@ Template['elements_balance'].helpers({
         
         if (EthTools.getUnit() === 'noether') return 'infinite';
 
+        var ethFormat = this.format || '0,0.00';
+
         if(balance){
             if(EthTools.getUnit() === 'usd' || EthTools.getUnit() === 'eur')
                 return EthTools.formatBalance(TemplateVar.get('balance'), '0,0.00');
             else if(EthTools.getUnit() === 'ether')
-                return EthTools.formatBalance(TemplateVar.get('balance'), '0,0.00[0000000000000000]');
+                return EthTools.formatBalance(TemplateVar.get('balance'), ethFormat);
```

Comment:
The default was already `0,0.00` so I'm not changing the default behavior, just adding a possibility of adding an extra parameter